.circleci/continue_config.yml

```diff
@@ -379,7 +379,7 @@ jobs:
           name: "Run Unit Tests"
           command: |
             cd presto-native-execution/_build/debug
-            ctest -j 8 -VV --output-on-failure --exclude-regex velox.*
+            ctest -j 8 -VV --output-on-failure --exclude-regex 'velox.*'
```

Comment:
I wonder if this can also be a problem with bash (even though it doesn't complain), due to how expansion works